### DISCUSSION TITLE
Adding a comment to issues/pr, then sending opening to send another one shows the text from the previous comment, and when pressing save, posts the same comment again

### DISCUSSION THREAD

**Discussion by GreeningSiren** [community/community#129699] [Mobile]
### Select Topic Area

Bug

### Body

Adding a comment to issues/pr, then sending opening to send another one shows the text from the previous comment, and when pressing save, posts the same comment again

I'm using an Android device with
GitHub Mobile v1.166.1 (10195)

**Comment by Iron-Ham** [comment]
Hello @GreeningSiren,

For additional context, can you provide some additional information? Can you include whether you're on iOS or Android, the version of iOS or Android you're using, as well as the version of the app you're experiencing this in? 


Thanks,
Hesham

**Comment by GreeningSiren** [comment]
Hello, 

I'm using an Android device with
GitHub Mobile v1.166.1 (10195)

**Comment by elisealix22** [comment]
@GreeningSiren taking a look at this and I just want to make sure these are the steps you are taking when seeing this!
- Open an issue
- Tap comment button
- Write a message
- Tap save
- Tap comment button again
- Write a second message
- Tap save

Is that correct? And when you tap save the second time the first message appears?

**Reply by GreeningSiren** [reply]
Hello,
Small correction. When you open the comment menu for the second time, the text from your first message shows up at the top, and when you press save it posts the part of the 1st comment again

**Reply by elisealix22** [reply]
Thanks for clarifying! We will look into this!

**Comment by xz-227** [comment]
# ?
